Lib/test/test_posix.py

```diff
@@ -1226,6 +1226,18 @@ def test_rename_dir_fd(self):
         finally:
             posix.close(f)
 
+    @unittest.skipUnless(hasattr(signal, 'SIGCHLD'), 'CLD_XXXX be placed in si_code for a SIGCHLD signal')
+    @unittest.skipUnless(hasattr(os, 'waitid_result'), "test needs os.waitid_result")
+    def test_cld_xxxx_constants(self):
+        # reference: http://man7.org/linux/man-pages/man2/sigaction.2.html
```

Comment:
These names are not Linux specific, and on other OSes they can have other values.

Reply:
> These names are not Linux specific, and on other OSes they can have other values.

I 've updated the unit test.
Please let me know if you have better ideas.